doc/libpmemobj/pmemobj_action.3.md

```diff
@@ -137,6 +138,13 @@ the transaction in which it is called. Only object reservations are supported
 in transactional publish. Once done, the reserved objects will follow normal
 transactional semantics. Can only be called during *TX_STAGE_WORK*.
 
+The **pmemobj_tx_xpublish**() function behaves exactly the same as
+**pmemobj_tx_publish**() when *flags* equals zero. *flags* is a
+bitmask of the following value:
+
++ **POBJ_XPUBLISH_NO_ABORT** - if the function ends incorrectly, return error
+number instead of aborting the transaction
```

Comment:
returning error number is orthogonal to aborting transaction - without jmp buffer this function will return an error and abort the transaction